plugins/GSdx/GSdx.rc

```diff
@@ -177,6 +177,33 @@ BEGIN
     GROUPBOX        "External Shader Settings",IDC_STATIC,6,138,236,88,BS_CENTER
 END
 
+IDD_OSD DIALOGEX 0, 0, 191, 229
+STYLE DS_SETFONT | DS_MODALFRAME | DS_FIXEDSYS | WS_POPUP | WS_CAPTION | WS_SYSMENU
+CAPTION "OSD Configuration"
+FONT 8, "MS Shell Dlg", 400, 0, 0x1
+BEGIN
+    DEFPUSHBUTTON   "OK",IDOK,42,209,50,14
+    DEFPUSHBUTTON   "Cancel",IDCANCEL,99,209,50,14
+    CONTROL         "Enable Log",IDC_OSD_LOG,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,15,6,54,10
+    CONTROL         "Enable Monitor",IDC_OSD_MONITOR,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,15,22,90,10
+    CONTROL         "Enable Indicator",IDC_OSD_INDICATOR,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,15,38,90,10
+    LTEXT           "Character size:",IDC_STATIC,15,57,120,8
+    EDITTEXT        IDC_OSD_SIZE_EDIT,15,68,58,14,ES_RIGHT | ES_AUTOHSCROLL
+    CONTROL         "",IDC_OSD_SIZE,"msctls_updown32",UDS_SETBUDDYINT | UDS_ALIGNRIGHT | UDS_AUTOBUDDY | UDS_ARROWKEYS | UDS_NOTHOUSANDS,15,69,11,14
+    LTEXT           "Scrolling speed:",IDC_STATIC,15,90,120,8
+    EDITTEXT        IDC_OSD_SPEED_EDIT,15,101,58,14,ES_RIGHT | ES_AUTOHSCROLL
+    CONTROL         "", IDC_OSD_SPEED,"msctls_updown32", UDS_SETBUDDYINT | UDS_ALIGNRIGHT | UDS_AUTOBUDDY | UDS_ARROWKEYS | UDS_NOTHOUSANDS,15,102,11,14
+    LTEXT           "Maximum number of onscreen messages:",IDC_STATIC,15,121,150,8
+    EDITTEXT        IDC_OSD_MAX_LOG_EDIT,15,132,58,14,ES_RIGHT | ES_AUTOHSCROLL
+    CONTROL         "", IDC_OSD_MAX_LOG,"msctls_updown32", UDS_SETBUDDYINT | UDS_ALIGNRIGHT | UDS_AUTOBUDDY | UDS_ARROWKEYS | UDS_NOTHOUSANDS,15,133,11,14
+    LTEXT           "Transparency:",IDC_STATIC,15,151,50,8	
+    CONTROL         "",IDC_OSD_TRANSPARENCY_SLIDER,"msctls_trackbar32",TBS_BOTH | TBS_NOTICKS | WS_TABSTOP,15,163,115,15
+    RTEXT           "100",IDC_OSD_TRANSPARENCY_TEXT,130,167,15,8
+    LTEXT           "Font:",IDC_OSD_FONT_TEXT,15,180,75,8
```

Comment:
IDC_OSD_FONT_TEXT doesn't seem to be used for anything. You could just use IDC_STATIC unless you plan to do something with it.

Apart from that, it seems fine, so squash the fixup into the appropriate commit. I'll merge after a few days if people are generally happy (or no one comments).